plugins/inputs/file/file_test.go

```diff
@@ -100,7 +103,8 @@ func TestGrokParser(t *testing.T) {
 	require.NoError(t, err)
 
 	err = r.Gather(&acc)
-	require.Equal(t, len(acc.Metrics), 2)
+	require.NoError(t, err)
+	require.Equal(t, 2, len(acc.Metrics))
```

Comment:
You should use
```suggestion
	require.Len(t, acc.Metrics, 2)
```
here.

Reply:
Done